addons/udes_stock/views/stock_picking.xml

```diff
@@ -98,7 +98,7 @@
                 Origin also set to readonly unless the picking is in draft state
             -->
             <xpath expr="//field[@name='origin']" position="attributes">
-                <attribute name="attrs">{'readonly': [('state', '!=', 'draft')],  'required': [('picking_type_code', '=', 'incoming')]}</attribute>
+                <attribute name="attrs">{'required': [('picking_type_code', '=', 'incoming')]}</attribute>
```

Comment:
Question: Does this mean that can be changed when picking is cancelled or done?
Do we want to be able to change on those states?